### PR TITLE
Playlist - 1

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Components - 
- Modal
- Playlist Modal

Features - 
- Create a playlist
- Add videos to the playlist
- Remove a video from playlists

### CODE REVIEW

File: src/backend/controllers/PlaylistController.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const addVideoToPlaylistHandler = function (schema, request) {
     const playlistId = request.params.playlistId;
     const { video } = JSON.parse(request.requestBody);
     const playlist = user.playlists.find((item) => item._id === playlistId);
-    if (playlist.videos.some((item) => item.id === video.id)) {
+    if (playlist.videos.some((item) => item._id === video._id)) {
```
Comment (rehanmohiuddin) [inline src/backend/controllers/PlaylistController.js:114]: can use ternary operator it looks clean

File: src/components/Modal/modal.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  position: fixed;
+  width: 30%;
+  max-width: 40rem;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 3rem;
+  background: $color-white;
+  z-index: 500;
+  border-radius: var(--border-radius-md);
```
Comment (ArjunGTX) [inline src/components/Modal/modal.scss:25]: why are some variables in css syntax and others in scss syntax?

**Review by rehanmohiuddin — APPROVED** [review]

**Review by ArjunGTX — APPROVED** [review]
Looks good

File: src/backend/controllers/PlaylistController.js (hunk 1 above)
Comment (SukanyaSen530) [reply src/backend/controllers/PlaylistController.js:114]: This is the backend playlist controller.

File: src/components/Modal/modal.scss (hunk 2 above)
Comment (SukanyaSen530) [reply src/components/Modal/modal.scss:25]: I have used my Component Library, there I am setting the default color and my components take it as normal css variables.

**Review by sharathnair9999 — APPROVED** [review]

**Pull request merged**